src/Illuminate/Contracts/Pagination/LengthAwarePaginator.php

```diff
@@ -26,4 +26,13 @@ public function total();
      * @return int
      */
     public function lastPage();
+
+    /**
+     * Get the URL for the last page.
+     *
+     * @author Stephen Damian <contact@devandweb.fr>
+     *
+     * @return string
+     */
+    public function lastPageUrl();
```

Comment:
Adding new methods to contracts on a minor/patch release is unfortunately a breaking change. Please remove this again from this PR.